cli/wallet/wallet_test.go

```diff
@@ -487,6 +487,57 @@ func TestWalletInit(t *testing.T) {
 					hex.EncodeToString(pubs[2].Bytes()),
 					hex.EncodeToString(pubs[3].Bytes()))...)
 			})
+
+			privs, pubs = testcli.GenerateKeys(t, 3)
+			script, err = smartcontract.CreateMultiSigRedeemScript(2, pubs)
+			require.NoError(t, err)
+			// Create a wallet and import a standard account
+			e.Run(t, "neo-go", "wallet", "init", "--wallet", walletPath)
+			e.In.WriteString("standardacc\rstdpass\rstdpass\r")
+			e.Run(t, "neo-go", "wallet", "import",
+				"--wallet", walletPath,
+				"--wif", privs[0].WIF())
+			w, err = wallet.NewWalletFromFile(walletPath)
+			require.NoError(t, err)
+			actual = w.GetAccount(privs[0].GetScriptHash())
+			require.NotNil(t, actual)
+			require.NotEqual(t, actual.Contract.Script, script)
```

Comment:
Don't need this check, we have a separate test for it.

Reply:
i was thinking about that this https://github.com/nspcc-dev/neo-go/pull/3293/commits/871a7d7a70d4d4ff91ecbf80de9ab38368834e34#diff-56abaed492595a64b9c0e27820f1e3ada1cc24d5d3a5c12829faf8f789308604R504 and 
 then this https://github.com/nspcc-dev/neo-go/pull/3293/commits/871a7d7a70d4d4ff91ecbf80de9ab38368834e34#diff-56abaed492595a64b9c0e27820f1e3ada1cc24d5d3a5c12829faf8f789308604R518 -shows that account truly converted to multisig and we check it

Reply:
OK.